packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -540,16 +558,35 @@ function attemptExplicitHydrationTarget(
 
 export function queueExplicitHydrationTarget(target: Node): void {
   if (enableSelectiveHydration) {
-    const priority = getCurrentPriorityLevel();
+    const schedulerPriority = getCurrentPriorityLevel();
+    const updateLanePriority = higherLanePriority(
+      getCurrentUpdateLanePriority(),
+      DefaultLanePriority,
+    );
     const queuedTarget: QueuedHydrationTarget = {
       blockedOn: null,
       target: target,
-      priority: priority,
+      priority: schedulerPriority,
+      lanePriority: updateLanePriority,
     };
     let i = 0;
-    for (; i < queuedExplicitHydrationTargets.length; i++) {
-      if (priority <= queuedExplicitHydrationTargets[i].priority) {
-        break;
+    if (enableCurrentUpdateLanePriority) {
```

Comment:
Same note as here: https://github.com/facebook/react/pull/19121/files/c533cb3335f4728f798d1b7fc2b7ad0847dfd02d#r445670773

The flag should only enable the warning. Shouldn't affect any behavior.